### PR TITLE
vehicle-attributes-recognition-barrier-0039 Update of accuracy values

### PR DESCRIPTION
**Pull request by elozgacx**
vehicle-attributes-recognition-barrier-0039 Update of accuracy values

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: models/intel/vehicle-attributes-recognition-barrier-0039/description/vehicle-attributes-recognition-barrier-0039.md [hunk 1]
```diff
@@ -17,15 +17,38 @@ This model presents a vehicle attributes classification algorithm for a traffic
...
 | Supported types       | Car, bus, truck, van                         |
-| GFlops                | 0.126                                        |
-| MParams               | 0.626                                        |
+| GFlops                | 0.13                                         |
+| MParams               | 0.6                                          |
 | Source framework      | Caffe*                                       |
 
 ## Accuracy - Confusion Matrix
 
-![](./vehicle-attributes-recognition-barrier-0039-2.png)
```
Comment (eaidova) [inline models/intel/vehicle-attributes-recognition-barrier-0039/description/vehicle-attributes-recognition-barrier-0039.md]: please return back removed image

File: models/intel/vehicle-attributes-recognition-barrier-0039/description/vehicle-attributes-recognition-barrier-0039.md [hunk 2]
```diff
@@ -46,4 +69,3 @@ Expected color order - BGR.
     [car, bus, truck, van]
 
 ## Legal Information
-[*] Other names and brands may be claimed as the property of others.
```
Comment (eaidova) [inline models/intel/vehicle-attributes-recognition-barrier-0039/description/vehicle-attributes-recognition-barrier-0039.md]: please return legal info string

File: models/intel/vehicle-attributes-recognition-barrier-0039/description/vehicle-attributes-recognition-barrier-0039.md (hunk 1 above)
Comment (eaidova) [reply models/intel/vehicle-attributes-recognition-barrier-0039/description/vehicle-attributes-recognition-barrier-0039.md]: olk. sory, I see, this image should be removed

**Comment by snosov1** [comment]
@DmitriySidnev please, review

**Comment by DmitriySidnev** [comment]
:+1:

**Comment by kchechil** [comment]
@zmaslova  please review GFlops/Mparams updates.

**Comment by kchechil** [comment]
Jenkins: ok to test

File: models/intel/vehicle-attributes-recognition-barrier-0039/description/vehicle-attributes-recognition-barrier-0039.md [hunk 3]
```diff
@@ -17,15 +17,38 @@ This model presents a vehicle attributes classification algorithm for a traffic
 | Min object width      | 72 pixels                                    |
 | Supported colors      | White, gray, yellow, red, green, blue, black |
 | Supported types       | Car, bus, truck, van                         |
-| GFlops                | 0.126                                        |
```
Comment (zmaslova) [inline models/intel/vehicle-attributes-recognition-barrier-0039/description/vehicle-attributes-recognition-barrier-0039.md]: why do you decrease accuracy? it's automated set values

**Pull request merged**